src/com/facebook/buck/android/AndroidResource.java

```diff
@@ -157,6 +158,16 @@
 
   private final AtomicReference<String> rDotJavaPackage;
 
+  private final Function<HasAndroidResourceDeps, Path> resourceDirectoryFunction =
+      new Function<HasAndroidResourceDeps, Path>() {
+        @Override
+        public Path apply(HasAndroidResourceDeps input) {
+          return Optional.fromNullable(input.getRes())
+              .transform(getResolver().getPathFunction())
+              .get();
+        }
+      };
+
```

Comment:
It occurred to me this morning that you'll also need to hook up any deps that are `android_resource` rules to the `android_resource` of the AAR (including reach inside any other AARs).  The easiest way to do that is likely to expose it on AndroidPrebuiltAAR.